src/_pytest/pytester.py

```diff
@@ -402,6 +402,12 @@ def __init__(self, ret, outlines, errlines, duration):
         self.stderr = LineMatcher(errlines)
         self.duration = duration
 
+    def __repr__(self):
+        return (
+            "<RunResult ret=%r len(stdout.lines)=%d len(stderr.lines)=%d duration=%.2fs>"
+            % (self.ret, len(self.stdout.lines), len(self.stderr.lines), self.duration)
```

Comment:
len appears to 1 always (empty line(s)), but appears to be ok?!

Reply:
I guess that's fine. 👍